modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -2680,10 +2704,16 @@ public boolean isActiveOnStart() {
      * @param activeOnStart Active on start flag value.
      * @return {@code this} instance.
      * @see #isActiveOnStart()
+     * @deprecated Use {@link #setClusterStateOnStart(ClusterState)} instead.
      */
+    @Deprecated
     public IgniteConfiguration setActiveOnStart(boolean activeOnStart) {
+        U.warn(log, "Property activeOnStart deprecated. Use clusterStateOnStart instead.");
```

Comment:
non informative here, who is **clusterStateOnStart** it ?

Reply:
Property of course) It's a public API merged in an Apache Ignite. So, I'd prefer to don't change int.

Reply:
this message non informative.

Reply:
I don't think so

Reply:
See my comment in a previous conversation.